samples/hello-navigation/src/androidTest/java/com/nhaarman/acorn/samples/hellonavigation/HelloNavigationTest.kt

```diff
@@ -22,8 +22,7 @@ import androidx.test.espresso.assertion.ViewAssertions.matches
 import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
 import androidx.test.espresso.matcher.ViewMatchers.withText
 import androidx.test.rule.ActivityTestRule
-import org.junit.Rule
-import org.junit.Test
+import org.junit.*
```

Comment:
**[ktlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Wildcard import (cannot be auto-corrected)